libraries/lib-numeric-formats/NumericConverter.cpp

```diff
@@ -70,418 +53,59 @@ static const TranslatableString BuildBeatsFormat() {
-    * rate! */
-   XO("01000,01000 frames|29.97002997")
-   },
-
-   {
-   /* i18n-hint: Name of time display format that shows time in hours, minutes,
-    * seconds and frames at PAL TV frame rate (used for European TV) */
-   { XO("hh:mm:ss + PAL frames (25 fps)") },
-   /* i18n-hint: Format string for displaying time in hours, minutes, seconds
-    * and frames with PAL TV frames. Change the 'h' to the abbreviation
-    * for hours, 'm' to the abbreviation for minutes, 's' to the abbreviation
-    * for seconds and translate 'frames'. Nice simple time code!
-    * The decimal separator is specified using '<' if your language uses a ',' or
-    * to '>' if your language uses a '.'. */
-   XO("0100 h 060 m 060 s+>25 frames")
-   },
-
-   {
-   /* i18n-hint: Name of time display format that shows time in frames at PAL
-    * TV frame rate (used for European TV) */
-   { XO("PAL frames (25 fps)") },
-   /* i18n-hint: Format string for displaying time in frames with NTSC frames.
-    * Change the comma
-    * in the middle to the 1000s separator for your locale, 
-    * translate 'frames' and leave the rest alone. */
-   XO("01000,01000 frames|25")
-   },
-
-   {
-   /* i18n-hint: Name of time display format that shows time in hours, minutes,
-    * seconds and frames at CD Audio frame rate (75 frames per second) */
-   { XO("hh:mm:ss + CDDA frames (75 fps)") },
-   /* i18n-hint: Format string for displaying time in hours, minutes, seconds
-    * and frames with CD Audio frames. Change the 'h' to the abbreviation
-    * for hours, 'm' to the abbreviation for minutes, 's' to the abbreviation
-    * for seconds and translate 'frames'.
-    * The decimal separator is specified using '<' if your language uses a ',' or
-    * to '>' if your language uses a '.'. */
-   XO("0100 h 060 m 060 s+>75 frames")
-   },
-
-   {
-   /* i18n-hint: Name of time display format that shows time in frames at CD
-    * Audio frame rate (75 frames per second) */
-   { XO("CDDA frames (75 fps)") },
-   /* i18n-hint: Format string for displaying time in frames with CD Audio
-    * frames. Change the comma
-    * in the middle to the 1000s separator for your locale, 
-    * translate 'frames' and leave the rest alone */
-   XO("01000,01000 frames|75")
-   },
-
-   // Beats and Measures must be the last format for UpdatePrefs to work
-   {
-   /* i18n-hint: Name of time display format that shows time beats and measures */
-   { XO("beats and measures") },
-   /* i18n-hint: Format string for displaying time in beats and measures.
-      * Look at the function for more detail */
-   BuildBeatsFormat()
-   },
-};
-
-namespace {
-struct BeatsUpdater : PrefsListener {
-   void UpdatePrefs() override
-   {
-      TimeConverterFormats_[WXSIZEOF(TimeConverterFormats_) - 1] =
-      {
-         { XO("beats and measures") },
-         BuildBeatsFormat()
-      };
-   }
-   BeatsUpdater() { UpdatePrefs(); }
-} theBeatsUpdater;
```

Comment:
This commit removes this without restoring an equivalent.  A later commit does that.  Stringly speaking it breaks a feature.  But I agree it's already too much to squash it all.